sphinx/testing/util.py

```diff
@@ -18,17 +19,16 @@
 import sphinx.application
 import sphinx.locale
 import sphinx.pycode
+from sphinx.util.console import strip_colors
```

Comment:
```suggestion
from sphinx.util.console import strip_colors as strip_escseq
```

Rationale: I think what the code is _intended_ to do may be to remove escape sequences.  What it actually does seems to be to remove ANSI color-code instructions.  Let's retain the original intend in the naming, rather than obscure/hide it.

(note: I don't know much about ANSI.  if this _does_ actually strip all escape sequences, then let's rename the function itself)

Reply:
Actually, it's not how we deprecate objects (see the `__getattr__` at the very end of the file). An no it does not strip all possible CSI.